src/digest/dune

```diff
@@ -0,0 +1,3 @@
+(library
+ (name digest)
```

Comment:
Can we prefix the new library names with `dune_`? That might matter for cleaner importing to our monorepo where all libraries live in one huge namespace =(

Reply:
I see. Surprised to hear you don't have the equivalent of dune's "scopes" for allowing private libraries to exist with whatever private name.

Reply:
Yeah, I'd love to have some scoping but we're not yet there.